src/Illuminate/Config/Attributes/InjectedConfig.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Illuminate\Config\Attributes;
+
+use Attribute;
+
+#[Attribute]
+class InjectedConfig
+{
+    public function __construct(string $key = '')
```

Comment:
```suggestion
    public function __construct(string $key)
```

The key seems to be required, defaulting it to an empty string doesn't make much sense IMO.